hw/index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Cubes</title>
+    <link rel="stylesheet" href="./src/css/style.css" type="text/css">
+</head>
+
+<body>
+    <section class="weatherWidget">
+        <div class="block first">
+            <picture class="weather">
+                <source srcset="src/img/CRAlt.svg" type="image/svg+xml">
```

Comment:
Good job! Using several sources and fallbacks is considered best practice: https://www.w3schools.com/tags/att_source_srcset.asp I suppose you plan to use different files as a default source and the fallback in the future?